kuma/static/js/helpfulness.js

```diff
@@ -4,6 +4,10 @@
     var articleTracker = doc.location.pathname + '#answered-helpful';
     // this feature requires localStorage
     if (win.mdn.features.localStorage) {
+        // clear out any drafts from localStorage, now that the document has been saved
```

Comment:
The helpfulness.js library is not included on the site anymore. 

This code would be better placed in wiki.js which is included on all the article pages by default.

Please put it on it's own lines with it's own check for `localStorage` rather than grouping it with unrelated actions. It can be the last thing in the file before the end of the closure statement.